### ISSUE TITLE
EMQX 5.0.18 - Swagger or OpenAPI version field missing in swagger.json

### ISSUE THREAD

**Issue by felixd** [emqx/emqx#10031]
### What happened?

https://HOST:18084/api-docs/swagger.json file is missing proper Swagger or OpenAPI version field.

https://HOST:18084/api-docs/index.html is not rendered properly.

![image](https://user-images.githubusercontent.com/4963164/221366932-5d05d71a-752c-4964-9d2e-d1754943b9c1.png)

```log
Unable to render this definition
The provided definition does not specify a valid version field.

Please indicate a valid Swagger or OpenAPI version field. Supported version fields are swagger: "2.0" and those that match openapi: 3.0.n (for example, openapi: 3.0.0).
```

### What did you expect to happen?

Proper Swagger API doc.

### How can we reproduce it (as minimally and precisely as possible)?

Install EMQX on Ubuntu 22 using offical documentation:

* https://www.emqx.com/en/blog/how-to-install-emqx-mqtt-broker-on-ubuntu

### EMQX version

<details>

```console
$ ./bin/emqx_ctl broker
sysdescr  : EMQX
version   : 5.0.18
datetime  : 2023-02-25T15:47:56.489356003+00:00
uptime    : 3 minutes, 51 seconds
```

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
PRETTY_NAME="Ubuntu 22.04.1 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.1 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy

$ uname -a
Linux xxxxx 5.15.0-1027-oracle #33-Ubuntu SMP Fri Jan 6 16:30:16 UTC 2023 aarch64 aarch64 aarch64 GNU/Linux

```
</details>

**Comment by felixd** [comment]
As a quick workaround I am using: https://raw.githubusercontent.com/emqx/emqx-docs/release-5.0/swagger/swagger.json as SWAGGER definition.

**Comment by zmstone** [comment]
Hi @felixd 
Thank you for the issue report
there seem to be a duplicated field definition `handshake_idle_timeout_ms`
will fix and release v5.0.19 tomorrow.

**Comment by zmstone** [comment]
the fix will be release in v5.0.19 (being tagged now)
